controllers/chat/moveToSign.js

```diff
@@ -18,7 +18,7 @@ const moveToSign = async (req, res) => {
 
   const client = StreamChat.getInstance(process.env.API_KEY, process.env.SECRET);
   try {
-    const {userModel, targetUserModel} = await Promise.allSettled([
+    const [userModel, targetUserModel] = await Promise.all([
       UsersFunction.findUserById(User, req?.userId),
       UsersFunction.findUserById(User, targetUserId)
     ]);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of `Promise.all()` instead of `Promise.allSettled()` could lead to potential issues if one of the promises fails. With `Promise.all()`, if any promise rejects, then the returned promise is immediately rejected with that reason. On the other hand, `Promise.allSettled()` waits for all promises to settle, regardless of whether they were fulfilled or rejected. This means that even if one promise fails, you still get the results of all the promises.

```diff
-    const [userModel, targetUserModel] = await Promise.all([
+    const [{value: userModel}, {value: targetUserModel}] = await Promise.allSettled([
       UsersFunction.findUserById(User, req?.userId),
       UsersFunction.findUserById(User, targetUserId)
     ]);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->